static/js/stream_edit.js

```diff
@@ -686,41 +582,6 @@ export function initialize() {
         return true;
     });
 
-    function save_stream_info(e) {
-        e.preventDefault();
-        e.stopPropagation();
-        const sub = get_sub_for_target(e.currentTarget);
-
-        const url = `/json/streams/${sub.stream_id}`;
-        const data = {};
-        const new_name = $("#change_stream_name").val().trim();
-        const new_description = $("#change_stream_description").val().trim();
-
-        if (new_name === sub.name && new_description === sub.description) {
-            return;
-        }
-        if (new_name !== sub.name) {
-            data.new_name = new_name;
-        }
-        if (new_description !== sub.description) {
-            data.description = new_description;
-        }
-
-        const $status_element = $(".stream_change_property_info");
-        dialog_widget.close_modal();
-        settings_ui.do_settings_change(channel.patch, url, data, $status_element);
```

Comment:
I think this code shoulc be removed in the commit which allows changing name and description in-place.